OpenDreamClient/Interface/DreamInterfaceManager.cs

```diff
@@ -357,25 +362,81 @@ sealed class DreamInterfaceManager : IDreamInterfaceManager {
             }
         }
 
+        public void WinClone(string controlId, string cloneId) {
+            ElementDescriptor elementDescriptor = null;
+            // window/pane
+            foreach (var windowDescriptor in InterfaceDescriptor.WindowDescriptors) {
+                if (windowDescriptor.Name == controlId) {
+                    elementDescriptor = windowDescriptor.WithName(_serializationManager, cloneId);
+                    ((WindowDescriptor)elementDescriptor).IsVisible = false; // per byond spec
```

Comment:
Make the `ControlWindow.WindowDescriptor` property public and use that instead of casting here.

Reply:
the `windowDescriptor` variable is already typed correctly, however due to the WithName method returning ElementDescriptor (a constraint caused by it being defined on the elementdescriptor type), the elementDescriptor variable has to be cast here